js/ToDoList.js

```diff
@@ -1,8 +1,15 @@
 import { Todoitem } from "./TodoItem .js";
 
 class ToDoList {
+  // btnMedeum = document.getElementById("btn-medeum");
+  // btnHigh = document.getElementById("btn-high");
+  // btnAll = document.getElementById("btn-all");
+
   constructor() {
     this.tasks = JSON.parse(localStorage.getItem("tasks")) || [];
+    this.lowTask = this.tasks.filter((task) => task.priority === "Low");
+    this.highTask = this.tasks.filter((task) => task.priority === "high");
+    this.medeumTask = this.tasks.filter((task) => task.priority === "Medeum");
```

Comment:
no need to **store** the tasks based on their priority, you just need to **show** the selected priority only, so you can manage that inside the render section